build_classlibs_osx.pl

```diff
@@ -40,7 +40,8 @@
 my $skipbuild=0;
 my $cleanbuild=1;
 my $jobs=8;
-my $sdk='10.6';
+my $sdk='10.11';
+my $macversion = '10.8';
```

Comment:
Don't we still support 10.6 or 10.7 in the standalone player? 

By setting -mmacosx-version-min=10.8, I think this means that that these builds won't work on OSX versions lower than 10.8. Each OSX SDK version supports deploying to older OSX versions, not sure how far OSX 10.11 SDK goes back.